lib/Service/OpenProjectAPIService.php

```diff
@@ -409,6 +411,9 @@ public function request(string $userId,
 				$response = $this->client->put($url, $options);
 			} elseif ($method === 'DELETE') {
 				$response = $this->client->delete($url, $options);
+				if ($response->getStatusCode() === 204) {
```

Comment:
use Http::STATUS_NO_CONTENT
```suggestion
				if ($response->getStatusCode() === Http::STATUS_NO_CONTENT) {
```